test/macs/cmac_test.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2013-present, the authors of the Pointy Castle project
+// This library is dually licensed under LGPL 3 and MPL 2.0.
+// See file LICENSE for more information.
+
+library pointycastle.test.macs.cmac_test;
+
+import "dart:typed_data";
+
+import "package:pointycastle/pointycastle.dart";
+
+import "../test/mac_tests.dart";
+
+void main() {
+  final mac = new Mac("AES/CMAC");
+
+  // Test vectors from AES-CMAC RFC, section 4.
+  // https://tools.ietf.org/html/rfc4493
+  final key = new Uint8List.fromList([
+    0x2b, 0x7e, 0x15, 0x16, 0x28, 0xae, 0xd2, 0xa6, //
+    0xab, 0xf7, 0x15, 0x88, 0x09, 0xcf, 0x4f, 0x3c, //
+  ]);
+  final keyParam = new KeyParameter(key);
+
+  mac.init(keyParam);
+
+  runMacTests(mac, [
+    PlainTextDigestPair(
+        Uint8List.fromList([]), "bb1d6929e95937287fa37d129b756746"),
+    PlainTextDigestPair(
+        Uint8List.fromList([
```

Comment:
I could change this to `createUint8ListFromString`?

Reply:
Whatever makes it easier to compare with the upstream vectors.